src/components/Timeline/Timeline.js

```diff
@@ -29,6 +29,19 @@ class Timeline extends Component {
     }
   }
 
+  shouldComponentUpdate (newProps) {
+    const { eventTypes, events, ticketId, incidentId, filter, history, dispatch } = newProps
+    if (this.props.ticketId !== newProps.ticketId) {
+      updatePagination(incidentId, dispatch)
```

Comment:
These actions are not giving the desired effect. The state we're losing is the events associated with the new incident… I believe what's occurring is we assume the events in the timeline will remain the same, and so when we click on the breadcrumb list it doesn't fetch new events. @pdimitratos Can we pair on this on Monday?

Reply:
Nevermind, I figured it out.